packages/primitives/src/Button.tsx

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { ComponentPropsWithRef, forwardRef } from "react";
+import { RecipeVariantProps, cva, cx } from "@ndla/styled-system/css";
+import { styled } from "@ndla/styled-system/jsx";
+
+export const buttonRecipe = cva({
```

Comment:
Sikkert en god grunn, men hvorfor har du gått for atomic isteden for config recipes?

Reply:
config recipes er avhengig av å tracke komponentbruk, og det gjøres v.h.a navn eller regex. Med andre ord vil `const StyledButton = styled(Button, {...})` ikke kunne trackes automatisk av panda. Tenkte det da var ryddigere å bare bruke cva.